src/tests/ingestion.int.test.js

```diff
@@ -0,0 +1,39 @@
+// Mocked import
+import childProcess from 'child_process' // eslint-disable-line import/order
+
+// Test imports
+import path from 'path'
+import { PassThrough } from 'stream'
+import FileIngestionEngine from '%src/components/ingestion/FileIngestionEngine'
+
+jest.mock('child_process')
+
+describe('ingestion', () => {
+	describe('FileIngestionEngine', () => {
+		it('it should read a video file into a Kafka topic', () => {
```

Comment:
My only thought is "should this integration test mock kafka / should we actually test that kafka is working?"

But I also think that could be a separate test that we write some day and we don't need to add that complexity to this PR.

Reply:
For instance should we at some point have an integration test that:

1. Ingests the file into kafka.
2. Reconstructs the file from kafka.
3. Ensures they're the same file.

Reply:
Good question! I wrestled with whether to mock that and ffmpeg. I think a test that does what you describe could be valuable, I also wonder whether the overhead of that will result in slower, brittler tests. But having coverage for the Kafka integration could be valuable. I think an interim solution is to add a little more specificity to what we’re expecting send to be called with, assuming send is going to work.

Reply:
Good call -- and maybe we'd want to get better with test tags before adding a test like that (so we could have `yarn test` default to just the fast ones and then like `yarn test:i:have:lots:of:time` for doing the true full suite...

Reply:
Made this test compare snapshots of the payloads passed to `send`.

Reply:
feature request that `yarn test:i:have:lots:of:time` loads up CATS (2019) dir. Tom Hooper in the background and takes the entire runtime to complete.

Reply:
Denied.